crypto/pem/pvkfmt.c

```diff
@@ -852,9 +859,9 @@ static int i2b_PVK(unsigned char **out, const EVP_PKEY *pk, int enclevel,
         if (!EVP_EncryptInit_ex(cctx, EVP_rc4(), NULL, keybuf, NULL))
             goto error;
         OPENSSL_cleanse(keybuf, 20);
-        if (!EVP_DecryptUpdate(cctx, p, &enctmplen, p, pklen - 8))
+        if (!EVP_EncryptUpdate(cctx, p, &enctmplen, p, pklen - 8))
```

Comment:
This is implying that this has never ever been called since it would have failed on the encrypt test??

Reply:
Well...  perhaps not in master.  In pre-3.0, it was more permissible to use EVP_DecryptUpdate instead of EVP_EncryptUpdate, because they're really the same there, so we got away with it.

I got hit by it when I added the tests for PVK in `test/serdes_test.c`